docker/academy-chain-compose.yml

```diff
@@ -16,7 +16,6 @@ services:
       - NODE_KEY_FILE=/data/$NODE01_ACCOUNT_ID/p2p_secret.txt
       - PORT=30333
       - RPC_PORT=9933
-      - WS_PORT=9944
```

Comment:
is both the http and ws connectivity now on the same port in this Substarte version? If yes then:

- make sure the first node (the one that exposes and binds ports to the host) uses whatever default ports the tools such as contratcs-ui or polkadot/js wallet expect (this used to be to ws on port 9944)
- remove the port that is not needed from the exposed and from the host mapping

Reply:
https://github.com/paritytech/substrate/pull/13384

Reply:
The default is now 9944 for both http and ws. Explained in https://github.com/paritytech/substrate/pull/12663#discussion_r1026563695 and confirmed by my own testing.